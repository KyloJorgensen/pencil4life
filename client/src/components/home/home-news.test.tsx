'use strict';

import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import HomeNews from './home-news';

let wrapper: ShallowWrapper;

describe('Render HomeNews', () => {
    it('should match the snaphot', () => {
        wrapper = shallow(
            <HomeNews/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
