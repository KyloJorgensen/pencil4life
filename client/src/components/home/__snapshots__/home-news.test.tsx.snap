// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HomeNews should match the snaphot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomeNews />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="selected"
          aria-current="page"
          exact={true}
          to="/news"
        >
          <h2>
            News
          </h2>
        </NavLink>,
        <UserConsumer
          limit={4}
        />,
        <NavLink
          activeClassName="selected"
          aria-current="page"
          exact={true}
          to="/news"
        >
          More News
        </NavLink>,
      ],
      "className": "news-wrapper p-2",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "selected",
          "aria-current": "page",
          "children": <h2>
            News
          </h2>,
          "exact": true,
          "to": "/news",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "News",
          },
          "ref": null,
          "rendered": "News",
          "type": "h2",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "limit": 4,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "selected",
          "aria-current": "page",
          "children": "More News",
          "exact": true,
          "to": "/news",
        },
        "ref": null,
        "rendered": "More News",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="selected"
            aria-current="page"
            exact={true}
            to="/news"
          >
            <h2>
              News
            </h2>
          </NavLink>,
          <UserConsumer
            limit={4}
          />,
          <NavLink
            activeClassName="selected"
            aria-current="page"
            exact={true}
            to="/news"
          >
            More News
          </NavLink>,
        ],
        "className": "news-wrapper p-2",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "selected",
            "aria-current": "page",
            "children": <h2>
              News
            </h2>,
            "exact": true,
            "to": "/news",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "News",
            },
            "ref": null,
            "rendered": "News",
            "type": "h2",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "limit": 4,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "selected",
            "aria-current": "page",
            "children": "More News",
            "exact": true,
            "to": "/news",
          },
          "ref": null,
          "rendered": "More News",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
